src/lib/components/blocks/TransactionCard.tsx

```
@@ -79,10 +79,20 @@ export const TransactionCard = ({ transaction }: TransactionCardProps) => {
             <code className="flex-1 truncate">
               {transaction.id.slice(0, 8)}...{transaction.id.slice(-8)}
             </code>
-            <button onClick={handleCopyHash} className="rounded p-1 hover:bg-accent" title="Copy transaction hash">
+            <button
+              onClick={handleCopyHash}
+              className="rounded p-1 transition-colors hover:bg-[color-mix(in_oklab,hsl(var(--background))_93%,hsl(var(--primary))_7%)] hover:text-[hsl(var(--button-hover-foreground))]"
+              title="Copy transaction hash"
+            >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)